### PR TITLE
Fix link to moved GBS specification

### PR DESCRIPTION
**Pull request by hacklschorsch**
This fixes a link I broke in #1366.

Refs ticket:4099 ([ticket](https://tahoe-lafs.org/trac/tahoe-lafs/ticket/4099))

### CODE REVIEW

**Review by meejah — APPROVED** [review]
More CI would pass if there was a newsfragment too

File: docs/specifications/url.rst [hunk 1]
```diff
@@ -40,7 +40,7 @@ NURLs
 
 The authentication and authorization properties of fURLs are a good fit for Tahoe-LAFS' requirements.
 These are not inherently tied to the Foolscap protocol itself.
-In particular they are beneficial to :doc:`../proposed/http-storage-node-protocol` which uses HTTP instead of Foolscap.
+In particular they are beneficial to :doc:`../specifications/http-storage-node-protocol` which uses HTTP instead of Foolscap.
```
Comment (meejah) [inline docs/specifications/url.rst:43]: hmm, might be simpler if this is just :doc:`http-storage-node-protocol` since it's in the right directory already?

**Comment by hacklschorsch** [comment]
Read the docs CI failed with `Failed to checkout revision: 74bbe4fe72c6aa6f06c9fd58ca2655597ddb5699` ???  Was it too fast for GitHub?

**Comment by meejah** [comment]
No changes / already merged? (Please re-open if that's not true)

**Comment by hacklschorsch** [comment]
You had fixed the same issue a day earlier in 4d34c775de03ef47b9af9903f26f6a6147fd3fa0 - don't know why I overlooked it.

**Pull request closed without merging**